src/registry/domain/repository.js

```diff
@@ -19,6 +19,10 @@ const errorToString = require('../../utils/error-to-string');
 module.exports = function(conf) {
   const cdn = !conf.local && new conf.storage.adapter(conf.storage.options);
   const options = !conf.local && conf.storage.options;
+  const storagePath =
```

Comment:
can we rather bring this logic here: https://github.com/opentable/oc/blob/master/src/registry/domain/options-sanitiser.js